ldap/servers/plugins/sync/sync_util.c

```diff
@@ -191,6 +191,14 @@ sync_create_state_control(Slapi_Entry *e, LDAPControl **ctrlp, int type, Sync_Co
     } else {
         slapi_entry_attr_find(e, SLAPI_ATTR_UNIQUEID, &attr);
         slapi_attr_first_value(attr, &val);
+        if ((attr == NULL) || (val == NULL)) {
```

Comment:
If attr is some here do we need to free it? Or is it owned by the entry?

Reply:
Thanks @Firstyear for your review. The attribute 'attr' or value 'val' are references into the entry 'e' they should not be freed here, later the operation cleanup will free them.